policybot/spanner.ddl

```diff
@@ -118,6 +118,10 @@ CREATE TABLE PullRequests (
   Assignees ARRAY<STRING(MAX)>,
   Title STRING(MAX) NOT NULL,
   Body STRING(MAX) NOT NULL,
+  BranchName STRING(MAX) NOT NULL,
```

Comment:
I just generate this file from Spanner, so the comments will be lost next time I update.